client/extensions/wp-super-cache/wrap-settings-form.jsx

```diff
@@ -142,6 +142,14 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 				wp_cache_path: '/wordpress/',
 				wp_cache_readonly: false,
 				wp_cache_writable: false,
+
+				// CDN
+				ossdl_cname: '',
+				ossdl_https: false,
+				ossdl_off_cdn_url: 'http://donnapep.wpsandbox.me',
```

Comment:
Yup, that's correct. This is just temporary until the next iteration which will involve hooking up to the REST API.